Documentation/config/postgres-database-cfg.yaml

```diff
@@ -0,0 +1,101 @@
+kind: List 
+metadata: {}
+apiVersion: v1
+items:
+  - apiVersion: image.openshift.io/v1
+    kind: ImageStream
+    metadata:
+      name: postgresql-10-rhel7
+    spec:
+      lookupPolicy:
+        local: true
+      tags:
+        - name: latest
+          from:
+            kind: DockerImage
+            name: registry.access.redhat.com/rhscl/postgresql-10-rhel7:latest
+          importPolicy: {}
+          referencePolicy:
+            type: Local
+  - apiVersion: v1
+    kind: Service
+    metadata:
+      name: postgresql
+    spec:
+      ports:
+        - name: postgresql
+          port: 5432
+          protocol: TCP
+          targetPort: 5432
+      selector:
+        name: postgresql
+  - kind: PersistentVolumeClaim
+    apiVersion: v1
+    metadata:
+      name: pgsql-data
+    spec:
+      accessModes:
+        - ReadWriteOnce
+      resources:
+        requests:
+          storage: 1Gi
+      storageClassName: ocs-storagecluster-cephfs
```

Comment:
storageClassName: `<default storage class name for cluster you are deploying on>`